### ISSUE TITLE
I have installed the pre-release 4.2 ModTek files into the mod directory, however when reinjecting the version still says 4.019688. (and BTML 6.4 is installed as well.)Yesterday, HBS sent a small update after the 1.3 releases. The game locks up when at the Battle loading screen planet. I verified game works fine. Previously, I have been running mods on 1.3 until after this update.

### ISSUE THREAD

**Issue by xeryx01** [janxious/ModTek#7]

**Comment by janxious** [comment]
You have to reinject BTML with the injector executable after any update from HBS.

**Comment by xeryx01** [comment]
Yes, I did do that and had it load the mods, it also sait that BTML was injected.  The problem is an incompatibility, that happens when running a mission. I even turned off all mods and tried it as well, and started a new campaign and missions will not run,

**Comment by xeryx01** [comment]
Again the first 1.3 worked fine, but 1.31 came out yesterday and it BROKE

**Comment by LadyAlekto** [comment]
You have to reinject BTML

If it were truly broke, the whole discord would know, since nobody has that, its user error

**Comment by xeryx01** [comment]
can you tell me what version number should be displayed, when the game boots up.

**Comment by LadyAlekto** [comment]
You got the latest modtek? then you got the right one

Dont try to overthink tiny things and cause user errors

**Comment by xeryx01** [comment]
yeah I downloaded the latest modtek.. 4.2 and btml 6.4

**Comment by xeryx01** [comment]
I had it all working fine until they updated the game to 131. Then i reinjected and ever since then the game stops at the mission screen.

**Comment by xeryx01** [comment]
i will reinstall btml, and modtek and see if it fixes itself

**Comment by xeryx01** [comment]
ok I found the problem

**Comment by xeryx01** [comment]
The assembly-Csharp.dll.orig was corrupted, so when I would restore it and reinjecty it kept a injecting into a bad file. I know this bcause I reinstalled all the modtek and btml files and it did not fix anything, infact it would not even re-inject properly, although it said it was injected. So I deleted the Csharp and reacquired through a steam verify of local files. Then reinjected. and finally it works
